### PR TITLE
Make more python code parse as python3.

### PR DESCRIPTION
**Pull request by Ms2ger**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/22273)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/324560367)

**Comment by Hexcles** [comment]
Did you use any automated tool?

**Comment by Ms2ger** [comment]
No, just ran the lint tool under python3.

**Comment by Hexcles** [comment]
Would it be a good idea to use some tools like 2to3 (and rely on six)?

**Comment by Ms2ger** [comment]
Everything here except for the `import` change will most likely be fixed the exact same way regardless of the approach we take for further changes, so I don't think we need to block on that decision.

**Comment by Hexcles** [comment]
Oh no. By no means that'd be a blocker. I thought you were still working in progress so suggested something to convert all the code.

Thanks.

**Comment by Ms2ger** [comment]
I will probably pick up more python3-related work in the future, but only when I have nothing better to do :)

File: mathml/tools/utils/misc.py [hunk 1]
```diff
@@ -13,14 +16,14 @@ def downloadWithProgressBar(url, outputDirectory="./", forceDownload=False):
...
-    request = urllib2.urlopen(url)
+    request = urlopen(url)
     totalSize = int(request.info().getheader('Content-Length').strip())
     bar = progressbar.ProgressBar(maxval=totalSize).start()
 
     chunkSize = 16 * 1024
     downloaded = 0
     print("Downloading %s" % url)
-    os.umask(0002)
+    os.umask(0o002)
```
Comment (foolip) [inline mathml/tools/utils/misc.py]: Looked a lot like cat on the keyboard, but apparently this is an octal literal: https://www.python.org/dev/peps/pep-3127/

File: cors/resources/status.py [hunk 2]
```diff
@@ -24,7 +24,7 @@ def main(request, response):
     if request.method == "OPTIONS" and "preflight" in request.GET:
         try:
             code = int(request.GET.first('preflight'))
-        except KeyError, ValueError:
+        except KeyError:
```
Comment (foolip) [inline cors/resources/status.py]: This looks like a mistake should ValueError really be removed? `int('bla')` throws ValueError. Should be `except (KeyError, ValueError)`?

**Review by foolip — APPROVED** [review]
LGTM % the `except (KeyError, ValueError)` case

File: cors/resources/status.py (hunk 2 above)
Comment (Ms2ger) [reply cors/resources/status.py]: That's not what the code did, though: in python 3, it'd be `except KeyError as ValueError`.

@jgraham, you wrote this code, thoughts? :)

Comment (foolip) [reply cors/resources/status.py]: Oh, wow, I guess it's a good thing that was changed in Python 3 :)

**Pull request merged**